test_util/integration_test.py

```diff
@@ -1340,3 +1344,55 @@ def wait_for_endpoint():
     r_data['Properties']['clusterId'] = ''
 
     assert r_data == exp_data
+
+
+class AgentManipulator:
+
+    def __init__(self, cluster, ssh_user, key_path):
+        self._cluster = cluster
+
+    def _run(self, cmd):
+        with closing(ssh_tunnel.SSHTunnel(self.ssh_user, self.key_path, self._cluster.host)) as tunnel:
+            tunnel.remote_cmd(cmd)
```

Comment:
If you want to do a one off command, use ssh.ssh_tunnel.run_remote_cmd which is literally identical to this